internal/support/notifications/errors/types.go

```diff
@@ -39,3 +39,31 @@ func (e ErrSubscriptionNotFound) Error() string {
 func NewErrSubscriptionNotFound(slug string) error {
 	return ErrSubscriptionNotFound{slug: slug}
 }
+
+type ErrMailAddrContainsCRLF struct {
+	slug             string
+	errMailAddresses []string
+}
+
+func (e ErrMailAddrContainsCRLF) Error() string {
+	return fmt.Sprintf("Subscription '%s' mail addresses contain CRLF %v", e.slug, e.errMailAddresses)
+}
+
+func (e *ErrMailAddrContainsCRLF) AddErrMailAddrContainsCRLF(m string) {
```

Comment:
Can we remove this method so that this error is more consistent with the others in the project? I think we can expect the code that wants to construct this error to be able to construct a slice of the offending email addresses and simply provide that to the constructor function.

Reply:
Thanks, I had spent couple minutes debating this myself. Done.